src/CanvasContainer.tsx

```diff
@@ -215,6 +219,64 @@ export const CanvasContainer: React.FC = ({ children }) => {
     };
   }, [canvasService]);
 
+  /**
+   * Keyboard events.
+   */
+  useEffect(() => {
+    function keydownListener(e: KeyboardEvent) {
+      const target = e.target as HTMLElement;
+
+      if (isTextInputLikeElement(target) || isTabLikeElement(target)) {
```

Comment:
This is, of course, nit-picking (to some extent) but it seems to me that if we want to have a bullet-proof logic that works in the very same way for different interactions then we should **selectively** apply those filters based on the pressed key.

Why this is IMHO important? If we allow, let's say, WSAD keys on buttons then why we don't allow them on tab elements? Tab elements don't have WSAD-based interactions of their own, after all. 

In addition to that - we should not handle arrow keys here when the target is an input of type range.